powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/UpdateRecoveryConfigRequest.java

```diff
@@ -28,10 +30,20 @@
 @Data
 public class UpdateRecoveryConfigRequest {
 
+    @Schema(description = "The identifier of the application")
+    @NotBlank(message = "Application ID must not be null when updating recovery code configuration")
     private String applicationId;
+
+    @Schema(description = "Whether recovery codes are enabled")
```

Comment:
I am not sure about this one though.
```suggestion
    @Schema(description = "Whether activation recovery is enabled")
```